app/main.ipynb

```diff
@@ -126,19 +126,20 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
     "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1])\n",
-    "# write your code here"
+    "\n",
+    "selected_subsequence = array[[1, 4, 5]]"
```

Comment:
The selected subsequence does not sum to 27 as required by the task. Ensure the correct indices are selected to achieve the desired sum.